infra/DockerRepoManager.py

```diff
@@ -0,0 +1,189 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Class to manage a git repos interaction with a docker image.
+
+This class is to be used to manage Oss-Fuzz projects docker build images.
+It allows for a build image to be build with a specific commit rather
+than just the current head of the repo.
+
+  Typical usage example:
+
+    drm = DockerRepoManager('curl')
+    current_commit = drm.get_image_commit()
+    drm.set_image_commit('df26f5f9c36e19cd503c0e462e9f72ad37b84c82')
+
+"""
+from helper import _build_image
```

Comment:
I think the functions prefixed with `_` are supposed to be "private" and shouldn't be called from any other module. If we need to re-use these, we should probably change the `helper.py` script too to give them regular names.

Reply:
+1. Please rename them in helper.py.

Reply:
+2 :-)
Also I would generally favor importing modules rather than importing the item from a module unless the length of `module.item` is long enough and is used frequently enough that importing directly helps readability.
Google's style guide even says never to import functions or classes from modules http://google.github.io/styleguide/pyguide.html#22-imports